modules/core_project_factory/variables.tf

```
@@ -71,6 +71,11 @@ variable "sa_role" {
   default     = ""
 }
 
+variable "apis_authority" {
+  description = "Defines which type of project_service to use: project_services/project_services_authority"
```

This description should be clarified:

```suggestion
  description = "Toggles authoritative management of project services."
```